packages/anagram.rb

```diff
@@ -4,10 +4,10 @@ class Anagram < Package
   description 'finds anagrams or permutations of words in the target phrase'
   homepage 'https://www.fourmilab.ch/anagram/'
   version '1.5'
-  license 'public-domain'
+  license 'CC0-1.0'
   compatibility 'all'
-  source_url 'https://www.fourmilab.ch/anagram/anagram-1.5.tar.gz'
-  source_sha256 '62eca59318782e889118a0e130d454e1c397aedd99fc59b2194393bf0eff5348'
+  source_url 'https://github.com/Fourmilab/anagram.git'
+  git_hashtag '5466ccd28c1b8f46bd8af420d17801bdf1da51a8'
```

Comment:
Definitely should rebuild the binaries to prove this update doesn't break the current package.

Reply:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/ffebbd80-f9a0-451a-987d-8147b3019bb6)

Reply:
Git version:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/49e431e4-f628-4e1a-b7f2-9fab7c9777a4)

Tar version:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/5467b8e2-1d75-469a-be8a-8968dc8d54d8)

Reply:
The old url is inconsistent-- I wasn't able to access it yesterday. Github will always work.

Reply:
Not sure what that means but whatever...